types/stylelint/index.d.ts

```diff
@@ -237,6 +255,7 @@ declare namespace stylelint {
 		configurationComment?: string | undefined;
 		fix?: boolean | undefined;
 		newline?: string | undefined;
+		lexer: Lexer;
```

Comment:
https://github.com/stylelint/stylelint/pull/8297#discussion_r1936022646

```suggestion
		lexer?: Lexer | undefined;
```